src/assets/scss/layout/_main.scss

```
@@ -37,6 +37,15 @@
 			.slack {
 				padding: 0.5em;
 			}
+			.major h2 {
+				text-align: center;
+			}
+			.major h3 {
+				font-size: 52px;
+			}
+			.major {

```

```suggestion
			.commands {
```

delete the two blocks above this for `.major h2` and `.major h3`